release.sh

```diff
@@ -86,16 +86,21 @@ if [ -z "$SPIN_GCS_BUCKET_PATH" ]; then
   SPIN_GCS_BUCKET_PATH=$PROD_SPIN_GCS_BUCKET_PATH
 fi
 
-for elem in darwin,amd64 linux,amd64; do
+for elem in darwin,amd64 linux,amd64 windows,amd64; do
```

Comment:
This looks good for this script, but you'd also want to add windows to this list as well to incorporate it into our buildtool: https://github.com/spinnaker/spinnaker/blob/master/dev/buildtool/spin_commands.py#L50.

Reply:
Sure thing, what's the difference in when each one is used for builds?

Reply:
This script is executed on a github release, and the other is executed nightly in our Jenkins instance to build the entirety of spinnaker via `buildtool` in spinnaker/spinnaker.

Reply:
Looking `spin_commands.py`, it seems a bit more involved. It appears to do things like check for the file `spin` in GCS, which would need to be updated to check for `spin.exe` for Windows, etc. Unfortunately, I've never used Python so it might take me a bit to figure it all out.

Reply:
Ah I see, hopefully should be relatively painless. Feel free to take a stab and reach out (me or @ewiseblatt) for help if you get stuck.